pom.xml

```diff
@@ -264,7 +264,6 @@
                 <!-- Fixes issue with tests failing in some environments. Replace this when proper version of 3.0 is released. -->
                 <version>3.0.0-M3</version>
                 <configuration>
-                    <argLine>-XX:+UseParNewGC</argLine>
```

Comment:
Whar are the consequences of removing this?

Reply:
That you can run the tests using java 11. Java 8 is long eol and getting hard to obtain.